apps/prairielearn/src/lib/code-caller/index.ts

```diff
@@ -70,7 +70,7 @@ export async function init() {
 
         await codeCaller.ensureChild();
         load.startJob('python_worker_idle', codeCaller.uuid);
-        return codeCaller;
+        return codeCaller as CodeCaller;
```

Comment:
Type inference should make this unnecessary.

```suggestion
        return codeCaller;
```